public_html/wp-content/mu-plugins/blocks/assets/src/shared/featured-image/inspector-control.js

```diff
@@ -43,47 +19,28 @@ class FeaturedImageInspectorControls extends Component {
 
 	constructor( props ) {
 		super( props );
-		this.availableSizes = sizePresets.map( ( size ) => size.value );
-	}
-
-	onPresetSizeSelect( size ) {
-		if ( size === '' ) {
-			return;
-		}
-		const { setAttributes } = this.props;
-		setAttributes( { featured_image_width: Number( size ) } );
 	}
 
 	render() {
-		const { attributes, setAttributes, title, help, selectLabel, cropLabel } = this.props;
+		const { attributes, setAttributes, title, help, selectLabel } = this.props;
 		const { featured_image_width } = attributes;
-		const selectedValue = this.availableSizes.indexOf( featured_image_width.toString() ) === -1 ? '' : featured_image_width.toString();
 		return (
 			<PanelBody
 				title={ title }
 				initialopen={ false }
 			>
 				<PanelRow>
```

Comment:
[SHOULD] `AvatarSizeControl` already has a `PanelBody`, so that and this `PanelRow` can probably be removed.

Reply:
fixed